integrationTests/multiShard/txScenarios/builtinFunctions_test.go

```diff
@@ -0,0 +1,119 @@
+package txScenarios
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTransaction_TransactionBuiltinFunctionsScenarios(t *testing.T) {
+	t.Parallel()
```

Comment:
I think there is an issue running this in parallel. For integration tests, might remove this.

Reply:
done.